README.md

```diff
@@ -0,0 +1,2 @@
+## SteemConnect4j  ![](https://travis-ci.org/hapramp/steemconnect4j.svg?branch=master)
```

Comment:
It must redirect to repo's travis location. Please use

`[![Build Status](https://travis-ci.org/hapramp/steemconnect4j.svg?branch=master)](https://travis-ci.org/hapramp/steemconnect4j)`